tool/ciphers.cc

```diff
@@ -21,19 +21,69 @@
 #include <openssl/ssl.h>
 
 #include "internal.h"
+#include "../ssl/internal.h"
 
+static const argument_t kArguments[] = {
+    {
+        "-openssl-name", kBooleanArgument,
+        "Print OpenSSL-style cipher names instead of IETF cipher names.",
+    },
+    {
+        "-cipher-query", kOptionalArgument,
+        "An OpenSSL-style cipher suite string that is matched against "
+        "supported ciphers. Defaults to \"ALL\".",
+    },
+    {
+        "-print-all", kBooleanArgument,
+        "Prints all supported AWS-LC libssl ciphers for all TLS versions. "
+        "If this option is used, all other options are ignored, except for "
+        "-openssl-name.",
+    },
```

Comment:
Is this any different than the above query with `ALL`?

Reply:
Yes, TLS 1.3 ciphers doesn't get added to the internal (run-time) cipher list there is queried below when doing the original cipher suite printing.

The NULL cipher is also not added. This PR just makes sure we print everything from the internal static list.